src/main/kotlin/com/stefan_grafberger/streamdq/anomalydetection/model/dto/NormalStrategyResultDto.kt

```diff
@@ -0,0 +1,6 @@
+package com.stefan_grafberger.streamdq.anomalydetection.model.dto
+
+data class NormalStrategyResultDto(val value: Double = 0.0,
```

Comment:
I think we use the Dto naming scheme nowhere else currently, do we? Would it be okay to rename this not to have the `Dto` in the name for consistency?